src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -277,7 +277,7 @@ export const WorkspaceList = ({ registeredUseCases$ }: WorkspaceListProps) => {
       width: '25%',
       truncateText: false,
       render: (lastUpdatedTime: string) => {
-        return moment(lastUpdatedTime).format('MMM DD[,]YYYY [@] HH:mm:ss.SSS');
+        return moment(lastUpdatedTime).format('MMM D, YYYY @ HH:mm:ss.SSS');
```

Comment:
In fact, I'm not suggesting using the same date format string with advanced setting. I'm prefer read the ui settings directly. Here is sample code: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/dashboard/public/application/utils/get_table_columns.tsx#L19 Feel free to have a try.

Reply:
okok, I see, thank you!